### DISCUSSION TITLE
Help with downloading file in browser  with ASP.NET

### DISCUSSION THREAD

**Discussion by martingrgv** [Tyrrrz/YoutubeExplode#751] [Q&A]
I am learning ASP.NET and decided to make myself a youtube mp3 converter but the problem that I am facing is that I don't know how to get the directory and make the browser to download it there. In the past with winforms I have used OpenFolderDialog. How can I implement this feature in my app.

**Comment by Tyrrrz** [comment]
You can use `FileResult`: https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.fileresult
